src/triage/component/catwalk/evaluation.py

```diff
@@ -438,6 +456,92 @@ def needs_evaluations(self, matrix_store, model_id, subset_hash=""):
         session.close()
         return needed
 
+    def _bias_audit(self,
```

Comment:
verbify name (e.g. _write_audit_to_db) to make it clear just from looking at the name that nothing is being returned.